deps/find_libpython.py

```diff
@@ -0,0 +1,376 @@
+#!/usr/bin/env python
+
+"""
+Locate libpython associated with this Python executable.
+"""
+
+# License
```

Comment:
I inlined the license so that it's easier to grab this file and put it in other projects.